app/src/main/java/org/systers/mentorship/view/adapters/AchievementsAdapter.kt

```diff
@@ -0,0 +1,60 @@
+package org.systers.mentorship.view.adapters
+
+import android.support.v7.recyclerview.extensions.ListAdapter
+import android.support.v7.util.DiffUtil
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.achievement_list_item.view.*
+import org.systers.mentorship.R
+import org.systers.mentorship.models.Task
+
+
+/**
+ * This class is the recyclerview adapter for achievements
+ * It is a subclass of [ListAdapter] for easy async calculation
+ * of diffs using DiffUtil to provide nice animations when
+ * the data set changes.
+ */
+class AchievementsAdapter : ListAdapter<Task, AchievementsAdapter.ViewHolder>(AchievementsItemCallback()) {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): AchievementsAdapter.ViewHolder {
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.achievement_list_item, parent, false)
+        return ViewHolder(view)
+    }
+
+    override fun onBindViewHolder(holder: AchievementsAdapter.ViewHolder, position: Int) {
+        holder.bind(getItem(position))
+    }
+
+    /**
+     * This class holds a view for each item of the Achievements list
+     * @param itemView represents each view of achievements list
+     */
+    inner class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+        /**
+         * This function binds the description of the achievement to the textview
+         *
+         * @param task The Achievement whose description is to be bound
+         */
+        fun bind(task: Task) {
+            itemView.tvDescription.text = task.description
+        }
+    }
+
+}
+
+/**
+ * This class represents the DiffCallback
+ * for tasks, and is used in [AchievementsAdapter]
+ */
+class AchievementsItemCallback : DiffUtil.ItemCallback<Task>() {
```

Comment:
I don't see where you are using this class and its functions. If this is from an old implementation of your's please remove this.

Reply:
This is the DiffUtil.ItemCallback implementation for the Task model class, and it is used in the achievements in the `AchievementsAdapter` class. It is a required parameter in the constructor. 

`class AchievementsAdapter : ListAdapter<Task, AchievementsAdapter.ViewHolder>(AchievementsItemCallback())`

ListAdapter requires an implementation like this to be passed to the constructor.